sssom_on_fhir/sssom_on_fhir.py

```diff
@@ -50,7 +50,8 @@ def convert_from_content_dir():
     the highest "v#", and convert any tsv files there, placing them in the content module\'s corresponding fhir/ dir."""
     # todo: temporarily skip icd10cm_snomed_from_omop and minimal_example
     content_modules: List[str] = os.listdir(CONTENT_DIR)
-    print()
+    
```

Comment:
## Implemented `--convert-from-content-dir`
This flag was here before but option was unimplemented.

> If this option is used, other CLI flags are ignored. This will go to the content/ dir look for all 'content module' directories there and, within each, look at tsv/, get the highest 'v#', and convert any tsv files there, placing them in the content module\'s corresponding fhir/ dir.